### DISCUSSION TITLE
plural key vs mnemonic tag for plural categories

### DISCUSSION THREAD

**Discussion by jpnjfk-sas** [unicode-org/message-format-wg#530] [Q&A] [answered]
Let me confirm if the plural key corresponds to [the mnemonic tag for plural categories](https://cldr.unicode.org/index/cldr-spec/plural-rules).
For example,

.match {$count :plural}
[one] {{You have one notification.}}
[few] {{You have few notifications.}}
[many] {{You have many notifications.}}
[ *] {{You have {$count} notifications.}}

The one, few, and many in the complex-message above correspond to pluralRule-count-one, pluralRule-count-few, pluralRule-count-many respectively defined in the [file](https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-core/supplemental/plurals.json)?
That means the meaning of one, few, and many depends on the locale.

**Comment by eemeli** [comment, marked as answer]
Yes, the meanings of the CLDR plural category keywords are locale-dependent.

**Reply by jpnjfk-sas** [reply]
Thank you for clarifying that!!
